M2/Macaulay2/packages/MultiprojectiveVarieties.m2

```diff
@@ -2182,6 +2423,9 @@ undocumented {
 (decompose,MultiprojectiveVariety),
```

Comment:
Most of these things should all be documented.  Observe that a documentation node may have multiple keys attached to it, so you may proceed to just add these keys to existing documentation nodes.

(The ones that don't need to be documented are the obvious ones, such as for net of a multiprojective variety.

Reply:
I have put them as undocumented because I consider them irrelevant additional functions and I don't want to draw attention to them. Some of them are already explained somewhere else. For instance, (symbol |,MultirationalMap,List) is explained in the documentation node (symbol |,MultirationalMap,MultiprojectiveVariety).

Reply:
It seems that `(symbol |,MultirationalMap,List)` would be a good thing to draw the user's attention to.  It might be useful.

Reply:
Okay, done!